src/components/widgets/sales-customers-engagement/sales-customers-engagement.tmpl.html

```diff
@@ -0,0 +1,30 @@
+<!--
+  Component generated by Impac! Widget Generator!
+-->
+<div widget-sales-customers-engagement>
+  <!-- Settings Panel -->
+  <div ng-show="widget.isEditMode" class="edit">
+    <h4>Widget settings</h4>
+
+    <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
+
+    <!-- Buttons displayed on the lower  -->
+    <div class="bottom-buttons" align="right">
+      <button class="btn btn-default" ng-click="initSettings()">Cancel</button>
+      <button class="btn btn-warning" ng-click="updateSettings()">Save</button>
+    </div>
+  </div>
+
+  <!-- Content Panel -->
+  <div ng-hide="widget.isEditMode">
+    <!-- Data found -->
+    <div ng-show="isChartDisplayed" class="">
```

Comment:
Does `isChartDisplayed` ever get set false?

Reply:
@manu-d as discussed could you check this?

Reply:
@manu-d ?